app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -203,6 +254,22 @@ function AcuantContextProvider({
 }: AcuantContextProviderProps) {
   const { isMobile } = useContext(DeviceContext);
   const { trackEvent } = useContext(AnalyticsContext);
+
+  // Set the appropriate SDK sources based on
+  // A/B Testing
+  const { acuantSdkUpgradeABTestingEnabled, useNewerSdk } = useContext(
+    AcuantSdkUpgradeABTestContext,
+  );
+  sdkSrc = getAcuantSdkSrc(sdkSrc, acuantSdkUpgradeABTestingEnabled, useNewerSdk);
+  cameraSrc = getAcuantCameraSrc(cameraSrc, acuantSdkUpgradeABTestingEnabled, useNewerSdk);
+
+  if (acuantSdkUpgradeABTestingEnabled) {
+    trackEvent('IdV: Acuant SDK Upgrade A/B Test', {
```

Comment:
This is a side-effect and shouldn't be in the top-level render path, it should be in a `useEffect` hook.